R/dataset.R

```diff
@@ -23,6 +23,7 @@ getNrow <- function (dataset) {
 #' Is it?
 #' @rdname crunch-is
 #' @param x an object
+#' @param value a value to assign
```

Comment:
Perhaps the solution then is to document `is.derived` on the same man page as `derivation`